### PR TITLE
Make sure the code completion takes both private imports and testable

### PR DESCRIPTION
**Pull request by aschwaighofer**
imports into account separately.

### CODE REVIEW

**Comment by aschwaighofer** [comment]
@swift-ci Please test

**Comment by aschwaighofer** [comment]
@swift-ci Please test

**Comment by aschwaighofer** [comment]
@swift-ci Please test

File: include/swift/AST/Module.h [hunk 1]
```diff
@@ -990,7 +990,18 @@ class SourceFile final : public FileUnit {
 
   void addImports(ArrayRef<ImportedModuleDesc> IM);
 
-  bool hasTestableOrPrivateImport(AccessLevel accessLevel, const ValueDecl *ofDecl) const;
+  enum ImportQueryKind {
+    /// Return the resuls for testable or private imports.
```
Comment (benlangmuir) [inline include/swift/AST/Module.h]: typo: missing "t" in results.

**Review by benlangmuir — CHANGES_REQUESTED** [review]
Please add a test case in test/IDE/complete_cache.swift. There's one for `@testable` that you could base it on.  Otherwise, LGTM!

**Comment by aschwaighofer** [comment]
@swift-ci Please smoke test

**Review by benlangmuir — APPROVED** [review]

**Pull request merged**